assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java

```diff
@@ -175,13 +175,15 @@ protected void configure() {
             "$HOME/che/terminal/che-websocket-terminal "
                 + "-addr :4411 "
                 + "-cmd ${SHELL_INTERPRETER} "
+                + "-enable-auth "
```

Comment:
Should not it be defined only in `assemlby-multiuser/assembly-wsmaster-war`? Have you checked that single user is working with your changes?

Reply:
yep, I'll rework ir

Reply:
I've moved binding to MultiUserCheWsMasterModule and CheWsMasterModule